_posts/2020-12-01-Fortran-Newsletter-December-2020.md

```diff
@@ -0,0 +1,149 @@
+* [#224](https://github.com/fortran-lang/fpm/pull/224): Add a reference document for the package manifest (fpm.toml).
+* [#221](https://github.com/fortran-lang/fpm/pull/221),
+  [#239](https://github.com/fortran-lang/fpm/pull/239): Runner options for test and app executables.
+* [#220](https://github.com/fortran-lang/fpm/pull/220): Implement compiler and flags settings in Haskell fpm.
+* [#209](https://github.com/fortran-lang/fpm/pull/209):
+  [#237](https://github.com/fortran-lang/fpm/pull/237): Developer API docs.
+* [#216](https://github.com/fortran-lang/fpm/pull/216),
+  [#225](https://github.com/fortran-lang/fpm/pull/225),
+  [#226](https://github.com/fortran-lang/fpm/pull/226),
+  [#229](https://github.com/fortran-lang/fpm/pull/229),
+  [#236](https://github.com/fortran-lang/fpm/pull/236),
+  [#240](https://github.com/fortran-lang/fpm/pull/240),
+  [#247](https://github.com/fortran-lang/fpm/pull/240): Other fixes and improvements.
+
+Work in progress:
+
+* [First beta release](https://github.com/fortran-lang/fpm/milestone/1) (WIP): First feature-complete release of the Fortran implementation.
+* (WIP) [#230](https://github.com/fortran-lang/fpm/pull/230),
+        [#261](https://github.com/fortran-lang/fpm/pull/261): Specification of the fpm CLI.
+* (WIP) [#232](https://github.com/fortran-lang/fpm/pull/232): Allowing the `extra` section in fpm.toml.
+* (WIP) [#248](https://github.com/fortran-lang/fpm/pull/248): Refactor backend for incremental rebuilds.
+* (WIP) [#251](https://github.com/fortran-lang/fpm/pull/251): Dependency management.
+* (WIP) [#255](https://github.com/fortran-lang/fpm/pull/255): Setting the compiler and specifying test or app target.
+* (WIP) [#257](https://github.com/fortran-lang/fpm/pull/257): Implement `fpm install`.
+* (WIP) [#260](https://github.com/fortran-lang/fpm/pull/260): Fix CI to test release build.
+
+fpm is still in early development and we need as much help as we can get.
+Here's how you can help today:
+
+* Use it and let us know what you think! Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md) to learn how to build your package with fpm, and the [manifest reference](https://github.com/fortran-lang/fpm/blob/master/manifest-reference.md) to learn what are all the things that you can specify in the fpm.toml file.
+* Browse the [open issues](https://github.com/fortran-lang/fpm/issues) and see if you can help implement any fixes or features.
+* Adapt your Fortran package for fpm and submit it to the [Registry](https://github.com/fortran-lang/fpm-registry).
+* Improve the documentation.
+
+The short-term goal of fpm is to make development and installation of Fortran packages with dependencies easier.
+Its long term goal is to build a rich and decentralized ecosystem of Fortran packages and create a healthy
+environment in which new open source Fortran projects are created and published with ease.
+
+## Compilers
+
+### Classic Flang
+
```

Comment:
We continue to evaluate and merge pull requests into Classic Flang. Recently merged pull requests into Classic Flang include:
* [PR#883: Flang generated executable does not show result variable of function](https://github.com/flang-compiler/flang/pull/883)
* [PR#933: Updating X-flag entries for internal command line option "-x 49"](https://github.com/flang-compiler/flang/pull/933)
* [PR#939: Publish Arm's internal documentation](https://github.com/flang-compiler/flang/pull/939)
* [PR#941: [DebugInfo] Internal subprogram variable is not accessible for printing in gdb](https://github.com/flang-compiler/flang/pull/941)
* [PR#942: Implement UNROLL(n) directive](https://github.com/flang-compiler/flang/pull/942)
* [PR#943: Enable github Actions for push to master and pull requests to master](https://github.com/flang-compiler/flang/pull/943)
* [PR#945: libpgmath: Stop using pgstdinit.h](https://github.com/flang-compiler/flang/pull/945)
* [PR#946: Call check_member() for PD_is_contiguous](https://github.com/flang-compiler/flang/pull/946)
* [PR#951: Fix for ICE in atomic instruction generation](https://github.com/flang-compiler/flang/pull/951)

Pull requests merged into the supporting projects:
* [classic flang LLVM monorepo PR#5: [Driver] Reduce downstream delta](https://github.com/flang-compiler/classic-flang-llvm-project/pull/5)
* [classic flang LLVM monorepo PR#6: Removing a few CI pipelines](https://github.com/flang-compiler/classic-flang-llvm-project/pull/6)
* [classic flang LLVM monorepo PR#7: Github Actions added to pre-compile artifacts for flang](https://github.com/flang-compiler/classic-flang-llvm-project/pull/7)
* [llvm mirror PR#87: Enable github actions for llvm](https://github.com/flang-compiler/llvm/pull/87)
* [flang-driver PR#94: Enable github actions](https://github.com/flang-compiler/flang-driver/pull/94)

The Classic Flang biweekly call has been set up to discuss issues and plans
for the next pull requests to be validated and merged. Our next calls are Wednesday, November 16 and 30, 8:00 AM Pacific time. The notes from previous calls, upcoming agenda and a link to join the call can be found
[here](https://docs.google.com/document/d/1-OuiKx4d7O6eLEJDBDKSRnSiUO2rgRR-c2Ga4AkrzOI).

Reply:
Thank you @gklimowicz. The call dates don't seem to be correct--can you check?

Reply:
Done. The dates were right, but the month was wrong. ;-)